{{cookiecutter.project_shortname}}/Dockerfile

```diff
@@ -20,12 +20,25 @@
 
 FROM inveniosoftware/centos7-python:3.6
 
+ARG include_assets
+
 COPY Pipfile Pipfile.lock ./
 RUN pipenv install --deploy --system
 
+COPY ./ .
 COPY ./docker/uwsgi/ ${INVENIO_INSTANCE_PATH}
 COPY ./invenio.cfg ${INVENIO_INSTANCE_PATH}
 COPY ./templates/ ${INVENIO_INSTANCE_PATH}/templates/
-COPY ./ .
```

Comment:
Normal: Why is this COPY being moved before? We would lose some of the caching.

Reply:
Fixed, sorry about that. Did not think about the caching and wanted to align with https://github.com/inveniosoftware/cookiecutter-invenio-instance/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/Dockerfile#L15, maybe we should port it back to them :) .